homeassistant/components/growatt_server/sensor/total.py

```diff
@@ -48,5 +50,6 @@
         api_key="nominalPower",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This one seems incorrect. As this is already a "max" value.

See: https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

> The state represents a measurement in present time, not a historical aggregation such as statistics or a prediction of the future.